hw/top_earlgrey/top_earlgrey.core

```diff
@@ -31,6 +32,15 @@ filesets:
       - ip/pinmux/rtl/autogen/pinmux_reg_pkg.sv
       - ip/pinmux/rtl/autogen/pinmux_reg_top.sv
       - ../ip/pinmux/rtl/pinmux.sv
+      - ip/alert_handler/rtl/autogen/alert_handler_reg_pkg.sv
+      - ip/alert_handler/rtl/autogen/alert_handler_reg_top.sv
+      - ../ip/alert_handler/rtl/alert_pkg.sv
```

Comment:
is this the same issue as pinmux? where the `alert_handler` core files contain regs, which actually need to be generated?

Do you want to try the other mechanism we discussed? :) Checking if the order is preserved in the dependency?

Reply:
it is the same issue yes. we need to solve this at some point (but I will not be able to try before I am back from vacation ;)).